### DISCUSSION TITLE
TBscrollキーマップの目的を教えてください

### DISCUSSION THREAD

**Discussion by koron** [Yowkees/keyball#25] [Q&A] [answered]
いまファームウェアのソースコードを整理(リファクタリング)しようとしていて
合わせてキーマップの方も整理するつもりでいます。
その際にviaやdefaultが用意されている理由はわかるのですが、
TBscrollに関してだけはわかりません。

どう直すか、場合によっては残すかどうかを検討したいので
なぜTBscrollというキーマップが用意されているのか
教えていただけないでしょうか?

**Comment by Yowkees** [comment, marked as answer]
TBscrollはビットトレードワンさんより販売されている1Uトラックボールを、Keyball右手のProMicro上に増設した際のキーマップになります。

一度試して動作はできましたが、しばらくは不要ですので一度削除して大丈夫です。

また必要となったときにDefaultを改造して作ろうと思います！

**Reply by koron** [reply]
なるほど。これですね。

https://shop.yushakobo.jp/products/adtb7m?variant=37665336164513

ではいったん削除する方向でやらせてもらいます。

**Reply by Yowkees** [reply]
はい。削除でお願いします。
よろしくお願いいたしますm(_ _)m
